README.md

```diff
@@ -1,59 +1,175 @@
+
+```bash
+kubectl gs template app \
+  --catalog giantswarm \
+  --name linkerd2-app \
+  --namespace linkerd \
+  --cluster <your-cluster-id>  \
+  --version 0.6.0 \
+  --user-configmap my-linkerd-values.yaml \
+  --user-secret my-linkerd-certificates.yaml \
+  --namespace-labels "linkerd.io/is-control-plane=true,config.linkerd.io/admission-webhooks=disabled,linkerd.io/control-plane-ns=linkerd" \
+  --namespace-annotations "linkerd.io/inject=disabled > linkerd-manifest.yaml"
+```
+
+The final `App` CR should look like this:
+
+```yaml
+...
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: linkerd2-app
+  namespace: <your-cluster-id>
+spec:
+  catalog: giantswarm
+  kubeConfig:
+    inCluster: false
+  name: linkerd2-app
+  namespace: linkerd
+  namespaceConfig:
+    annotations:
+      linkerd.io/inject: disabled
+    labels:
+      config.linkerd.io/admission-webhooks: disabled
+      linkerd.io/control-plane-ns: linkerd
+      linkerd.io/is-control-plane: "true"
+  userConfig:
+    configMap:
+      name: linkerd2-app-userconfig-<your-cluster-id>
+      namespace: <your-cluster-id>
+    secret:
+      name: linkerd2-app-userconfig-<your-cluster-id>
+      namespace: <your-cluster-id>
+  version: 0.6.0
+```
 
-This chart is based on the official linkerd2 helm chart. The main difference it offers is a
-direct integration with [cert-manager](https://cert-manager.io/) used to generate short-lived
-certificates for linkerd2 components. This results in an entirely hands-off install - no
-need to generate a CA certificate prior to installation.
+Please note, that for security reasons Giant Swarm by default forbids the usage of
+`emptyDir` volumes as storage for pods. Linkerd needs this functionality to deploy
+linkerd itself, but also to run any deployment that is going to be included in
+the service mesh. Enabling `emptyDir` volumes poses a risk that a pod will create
+such big `emptyDir` that the underlying cluster node will run out of disk space.
+If you're OK with this potential issue, the easiest way to allow for `emptyDir`
+for all the deployments is to edit the default PSP of your cluster by running the
+command:
+
+```bash
+kubectl edit psp restricted
+```
 
-## Requirements
+then finding the `spec.volumes` list and appending `emptyDir` value to the list. Then, write the file and exit your editor.
 
-- you can install only one release of this chart per kubernetes cluster.
-- `cert-manager` must already be deployed in the cluster.
-- it is strongly suggested to use the [`linkerd2-cni-app`](https://github.com/giantswarm/linkerd2-cni-app) as this results in a more secure setup.
-  - the CNI must be installed before this chart, and `global.cniEnabled: true` must be set.
+To finally deploy linkerd, run
```

Comment:
The chart includes an own PSP, so for the bare app to be installed, the `restricted` psp does not need to be edited, so IMO this `finally` could be a little confusing